src/Mod/Spreadsheet/Gui/SheetModel.cpp

```diff
@@ -63,6 +63,27 @@ SheetModel::SheetModel(Sheet* _sheet, QObject* parent)
         QColor(QString::fromStdString(hGrp->GetASCII("PositiveNumberColor", "#000000")));
     negativeFgColor =
         QColor(QString::fromStdString(hGrp->GetASCII("NegativeNumberColor", "#000000")));
+
+
+    const QStringList alphabet{QStringLiteral("A"), QStringLiteral("B"), QStringLiteral("C"),
+                              QStringLiteral("D"), QStringLiteral("E"), QStringLiteral("F"),
+                              QStringLiteral("G"), QStringLiteral("H"), QStringLiteral("I"),
+                              QStringLiteral("J"), QStringLiteral("K"), QStringLiteral("L"),
+                              QStringLiteral("M"), QStringLiteral("N"), QStringLiteral("O"),
+                              QStringLiteral("P"), QStringLiteral("Q"), QStringLiteral("R"),
+                              QStringLiteral("S"), QStringLiteral("T"), QStringLiteral("U"),
+                              QStringLiteral("V"), QStringLiteral("W"), QStringLiteral("X"),
+                              QStringLiteral("Y"), QStringLiteral("Z")};
+
+    for (const QString &letter : alphabet)
+        columnLabels << letter;
+
+    for (const QString &left : alphabet)
+        for (const QString &right : alphabet)
+            columnLabels << left + right;
+
+    for (int i = 1; i < 16385; i++)
```

Comment:
It would be good to extract 16385 as constexpr - either in method or in class. Something like 
```
constexpr size_t maxRowCount = 16385;
```

would do the job - it would be clear what that number means.